lib/build-database.js

```diff
@@ -316,17 +325,26 @@ const importLines = async trx => {
               line_id: line.id,
               source_id: sourceId,
               gurmukhi,
-              first_letters: firstLetters( gurmukhi ),
-              vishraam_first_letters: firstLetters( gurmukhi, true, true ),
+              first_letters: [
+                stripEndings,
+                stripVishraams,
+                firstLetters,
+              ].reduce( ( text, fn ) => fn( text ), gurmukhi ),
+              vishraam_first_letters: [
+                stripEndings,
+                // Retain heavy vishraams only
+                text => stripVishraams( text, { light: true, medium: true } ),
```

Comment:
it's odd that you have to choose what to strip (I'd expected all weights by default and code having to toggle `heavy: false`)

Reply:
All weights are stripped by default, but if you supply options, then you choose which are (because I thought it'd more be confusing to specify which you'd exclude in a strip function)

Reply:
I think it makes more sense to change parameters to affect the default. So if the default is light, medium, heavy true. You'd put in a parameter to make one of these false. But if you think this makes more sense to people, that's fine -- I am not up to date on these conventions lol